### PR TITLE
Annotate ServiceAccount in Helm chart and add commented-out examples in values.yaml

### PR DESCRIPTION
**Pull request by jayg2309**
With reference to issue #958 

1) it would be nice if you could annotate the serviceaccount via the helm chart rather than creating a seperate service account for workload identity
Screenshot 2024-10-25 215611

2) also, it would be nice if the values.yaml in git had some commented-out examples for configuration: secret: etc
Screenshot 2024-10-25 215625
Screenshot 2024-10-25 215639

@anna-geller Please have a look

### CODE REVIEW

File: content/docs/02.installation/05.kubernetes-gcp-gke.md [hunk 1]
```diff
@@ -59,6 +59,29 @@ helm repo add kestra https://helm.kestra.io/
 helm install my-kestra kestra/kestra
 ```
 
+**Workload Identity Setup(Optional)**
```
Comment (anna-geller) [inline content/docs/02.installation/05.kubernetes-gcp-gke.md:62]:
```suggestion
## Workload Identity Setup
```

File: content/docs/02.installation/05.kubernetes-gcp-gke.md [hunk 2]
```diff
@@ -59,6 +59,29 @@ helm repo add kestra https://helm.kestra.io/
...
+
+If you are using Google Cloud Workload Identity, you can annotate your Kubernetes service account in the Helm chart configuration. This will allow Kestra to automatically use the associated GCP service account for authentication.
```
Comment (anna-geller) [inline content/docs/02.installation/05.kubernetes-gcp-gke.md:64]: Did you test that this works? if so, can you add a couple of screenshots?

Comment (jayg2309) [reply content/docs/02.installation/05.kubernetes-gcp-gke.md:64]: I haven't tested it since GKE is not free.

Comment (anna-geller) [reply content/docs/02.installation/05.kubernetes-gcp-gke.md:64]: no worries

**Pull request merged**